src/hooks/useBifrostWebSocketEffect.ts

```diff
@@ -40,8 +43,21 @@ interface IWsResponseData {
 
 interface IUseBifrostWebSocketEffectOptions extends IUsePncWebSocketEffectOptions {
   buildId?: string;
+  closeResultId?: string;
 }
 
+const getPrefixFilters = (buildId?: string, closeResultId?: string) => {
```

Comment:
I would consider one object parameter of all the ID values; same for `getMatchFilters`

Reply:
@patrikk0123 The reason for having `getPrefixFilters` and `getMatchFilters` is to use a return style to replace a complex if-else structure. Altogether we have 4 kinds of logs: build logs, milestone close result logs, deliverables analysis logs and brew push logs. especially we need both buildId and brewPushId for brew push logs. We will need to pass different params to handle it as we don't know what will be passed from the front-end.

Reply:
i just meant using this syntax:
```tsx
({ buildId, closeResultId }: { buildId?: string, closeResultId?: string} ) => {}
```

which is more neat especially when there are 4 log variants